lutris/services/egs.py

```diff
@@ -169,8 +167,8 @@ class EpicGamesStoreService(OnlineService):
         'Chrome/84.0.4147.38 Safari/537.36'
     )
 
-    def __init__(self):
-        super().__init__()
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
```

Comment:
Strange to use *args, **kwargs here. The base class wants just 'id', the service id.

More readable to have the argument list explicit, I say.